test/integration/garbage_collector_test.go

```diff
@@ -357,7 +359,7 @@ func TestStressingCascadingDeletion(t *testing.T) {
 	go gc.Run(5, stopCh)
 	defer close(stopCh)
 
-	const collections = 30
+	const collections = 10
```

Comment:
30 collections is creating 120 rc and 480 pods and then deleting 120 rcs. Does that sound correct to take 6mins?